### DISCUSSION TITLE
Starting development server always fails

### DISCUSSION THREAD

**Discussion by MaximilianGaedke** [resend/react-email#661] [Q&A]
What can I do to solve the problem? I tried it both with the manual and with the automatic setup and it always stopped. I also have the latest Node.js version.

**Comment by bukinoshita** [comment]
What's the error you're getting?

**Reply by gfellah45** [reply]
Currently having the same issue. The local dev server not running. 

appreciate any help

Thanks

<img width="639" alt="reactemail" src="https://user-images.githubusercontent.com/20698003/234698072-5d6fd145-b49e-4582-94fb-4410924ec54e.PNG">

**Reply by MaximilianGaedke** [reply]
I get no error, that is the point :(

**Reply by bukinoshita** [reply]
Do you guys have any reproduction repository so I can take a look?

**Reply by MaximilianGaedke** [reply]
For sure: https://github.com/MaximilianGaedke/react-email-test

**Reply by MaximilianGaedke** [reply]
It´s not really complicated and I also didn´t added any code to it so that´s why I´m so surprised...

**Comment by jinsley8** [comment]
I can't get `email dev` to work either in a Next.js app.

I have `/emails` folder and when I run `yarn email dev` it generates a new `.react-email` folder and hangs up at the [2/5] part here:

```
✔ Emails preview generated
⠋ Installing dependencies...
warning ../../../../../package.json: No license field
[1/5] Validating package.json...
[2/5] Resolving packages...
```

If I visit `localhost:3000` I get a runtime error:
```
Error: ENOENT: no such file or directory, scandir '/project-name/.react-email/emails'
```

Is there any other way to preview emails without running this?
